samples/README.md

```diff
@@ -4,65 +4,171 @@ This repository contains a few samples to help you get started quickly with the
 
-A sample Python consumer function is provided in samples/python/KafkaTrigger. It depends on the Kafka installed locally (localhost:9092), as described in previous section.
+For more details, refer to [Local settings file](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Ccsharp%2Cbash#local-settings-file).
 
-### Using the Azure Functions Python Kakfa Trigger
+```json
+{
+    "IsEncrypted": false,
+    "Values": {
+        "BrokerList": "pkc-epwny.eastus.azure.confluent.cloud:9092",
+        "ConfluentCloudUsername": "WHYYKOT6JMPYGJJV",
+        "ConfluentCloudPassword": "KUtytGe72JzhCKeA7cZ6dJj6oRE4I5NyOjMxj157tQaiJoVy67UD0sAqJM9e4QAZ",
+        "FUNCTIONS_WORKER_RUNTIME": "python",
+        "AzureWebJobsStorage": ""
+    }
+}
+```
 
-1. Make sure you have access to a Kafka Cluster. Follow [these](https://medium.com/@tsuyoshiushio/local-kafka-cluster-on-kubernetes-on-your-pc-in-5-minutes-651a2ff4dcde) steps to set it up locally
+### Extension Bundle and install kafka extension
 
-2. Make sure you have [latest version](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local) of functions core tools.
-[TODO: add information Java samples]
+For the latest Azure Function, we use [Extension Bundle](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-register#extension-bundles). However, for the Kafka extension, we are currently doesn't use it. In the near feature, Kafka extension will support it. The extnsion bundle will install extension automatically. So that we need to install Kafka extension manually.
 
-### Using the Auzre Functions Java kafka Trigger
+For installing Kafka extension manually:
 
-Go to [Developers guid for Kafka Functions for Java](java/README.MD). You can find the step-by-step instruction.
+#### Create extensions.csproj file
 
-## Typescript Consumer
+Azure Functions extension is written in C#. We need to install with .NET Core capability. `csproj` file is a project file for .NET. For more information refer to [Understanding the project file](https://docs.microsoft.com/en-us/aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/understanding-the-project-file)
 
-A sample Typescript function is provided in samples/typescript/kafka-trigger. It can be executed locally or using [Visual Studio Code Remote Containers](https://code.visualstudio.com/docs/remote/remote-overview):
+_extensions.csproj_
 
-### Locally
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+	<WarningsAsErrors></WarningsAsErrors>
+	<DefaultItemExcludes>**</DefaultItemExcludes>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Kafka" Version="2.0.0-beta" />
```

Comment:
We need to update it to the latest release in the next pr.

Reply:
Absolutely!